### PR TITLE
Minor fixes for Distribution release v202211

### PR DESCRIPTION
**Pull request by IshaanDesai**
This PR fixes minor things mentioned in https://github.com/precice/tutorials/pull/305#issuecomment-1313828439 as part of the Distribution release v202211.

### CODE REVIEW

**Comment by IshaanDesai** [comment]
To be merged after https://github.com/precice/tutorials/pull/307

File: pages/tutorials/tutorials.md [hunk 1]
```diff
@@ -49,6 +49,7 @@ In the following cases, you can explore different aspects of preCICE:
 - [Volume-coupled diffusion](tutorials-volume-coupled-diffusion.html): An experimental volume coupling scenario, with two FEniCS solvers.
 - [ASTE turbine](tutorials-aste-turbine.html): An example case for ASTE to investigate different preCICE mappings using a turbine geometry.
 - [Channel transport](tutorials-channel-transport.html): A channel flow coupled to a transport (of, e.g., a chemistry species) in a uni-directional way, with Nutils.
+- [Channel transport reaction](tutorials-channel-transport-reaction.html): A channel flow coupled to a transport of advection-diffusion type (of, e.g., a chemistry species) in a uni-directional way, with FEniCS.
```
Comment (uekerman) [inline pages/tutorials/tutorials.md]:
```suggestion
- [Channel transport reaction](tutorials-channel-transport-reaction.html): A channel flow coupled to a transport of a chemistry species with reaction in a uni-directional way, with FEniCS.
```

File: _config.yml [hunk 2]
```diff
@@ -207,6 +208,7 @@ subprojects:
   - imported/tutorials/partitioned-pipe
   - imported/tutorials/volume-coupled-diffusion
   - imported/tutorials/channel-transport
+  - imported/tutorials/channel-transport-reaction
```
Comment (uekerman) [inline _config.yml:211]: Aren't these files auto generated? @MakisH ?

Comment (MakisH) [reply _config.yml:211]: The files are imported as a git module, but we do need to specify them here.

**Review by MakisH — APPROVED** [review]
Looks good, I have not tested it, though.

**Pull request merged**